src/app/dataset-view/dataset.component.ts

```diff
@@ -206,6 +217,32 @@ const language = 'typescript';
         );
     }
 
+    public getTitle(node: any): string {
+        if (node.event.__typename === "MetadataEventAddData") {
```

Comment:
Perhaps switch statement would be less verbose?
https://www.w3schools.com/jsref/jsref_switch.asp